next-env.d.ts

```diff
@@ -1,8 +1,3 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: MPL-2.0
- */
-
```

Comment:
is the bot gonna get angry with us for removing this?

Reply:
Probably 👀 I think we can define some config to exclude this file, I'll do that in a follow up!

Reply:
https://github.com/hashicorp/react-components/pull/898